docs/ref/models/querysets.txt

```diff
@@ -2065,6 +2065,8 @@ are)::
     ...     Entry(headline='This is only a test'),
     ... ])
 
+Returns ``objs``.
```

Comment:
It is mentioned in the caveats below, but this is `objs` coerced to a `list`, and each object will only have the `pk` set if the query was against PostgreSQL.

I don't know whether we want to be clearer and state that it a modified version of `objs` in accordance with the caveats below?

Reply:
I moved it to the bottom, so it should be read after the caveats and description of the arguments, and added wording to indicate it's cast to a list.